arrow-libs/optics/arrow-optics/src/main/kotlin/arrow/optics/typeclasses/Snoc.kt

```diff
@@ -27,12 +33,12 @@ fun interface Snoc<S, A> {
   /**
    * Provides an [Optional] between [S] and its init [S].
    */
-  fun initOption(): Optional<S, S> = snoc() compose Tuple2.first()
+  fun initOption(): Optional<S, S> = snoc() compose PLens.tuple2First()
 
   /**
    * Provides an [Optional] between [S] and its last element [A].
    */
-  fun lastOption(): Optional<S, A> = snoc() compose Tuple2.second()
+  fun lastOption(): Optional<S, A> = snoc() compose PLens.tuple2Second()
```

Comment:
This can stay as it is until `0.13.0`

Reply:
thanks @nomisRev , I hope I addressed all the comments! 😄